recipes/angsd/build.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+binaries="\
+misc/msToGlf \
+misc/contamination \
+misc/contamination2 \
+misc/splitgl \
+misc/thetaStat \
+misc/printIcounts \
+misc/NGSadmix \
+misc/realSFS \
+misc/smartCount \
+misc/supersim \
+angsd \
+"
+BINDIR=$PREFIX/bin
+mkdir -p $BINDIR
+
+
+make LDFLAGS="-L=${PREFIX}/lib" INC="-I=${PREFIX}/include"
```

Comment:
Could you use standard env vars like `CPLUS_INCLUDE` or `C_INCLUDE`. That could be a way to avoid the makefile patches.